package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "yarn pretty && eslint --max-warnings 0 --ext ts,tsx,json src"
   },
   "dependencies": {
-    "@big-whale-labs/constants": "^0.1.7",
+    "@big-whale-labs/constants": "^0.1.16",
```

Comment:
It's the latest version: https://www.npmjs.com/package/@big-whale-labs/constants